articleController.js

```diff
@@ -1,5 +1,19 @@
 var articleController = {};
 
 articleController.index = function () {
-  blog.determineData(); 
+  blog.determineData();
+};
+
+articleController.category = function (ctx, next) {
+  var categoryData = function (data) { //rename data to articles?  it is the return of .findByCategory
+    ctx.articles = data;
+    next();
+  };
+  CompleteArticle.findByCategory(ctx.params.category, categoryData);
+};
+
+articleController.show = function(ctx, next) {
+  blog.loadBlogPage();
```

Comment:
I had to really do some detective work here, but it looks like your `blog.loadBlogPage();` function doesn't have a way to actually do anything with these articles that you are loading, and that's why the page isn't displaying anything. After making a few changes I was able to see the `ctx.articles` was populated with our articles, but now you need to find a way to display them.